isyntax2raw/cli/isyntax2raw.py

```diff
@@ -18,7 +18,7 @@ def cli():
     pass
 
 
-@click.command()
+@cli.command(name='write_tiles')

```

Comment:
Did we need to do this vs. just using `@click.command()`?

Reply:
https://click.palletsprojects.com/en/8.0.x/commands/

https://github.com/pallets/click/issues/1123